.github/workflows/release.yml

```diff
@@ -11,11 +11,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
```

Comment:
Quick questions: @Dawntraoz I was also configuring release.yml. I see that you set `actions/checkout@v3` and `actions/setup-node@v3`. Alba and @alexjoverm Should we align with v3 in all SDKs or what's the difference?